src/test/kotlin/ApplicationLocalPostgres.kt

```diff
@@ -13,10 +13,10 @@ fun main(args: Array<String>) {
     val properties = Properties()
     properties["DATASOURCE_URL"] = "jdbc:postgresql://localhost:5432/familie-ef-sak"
     properties["DATASOURCE_USERNAME"] = "postgres"
-    properties["DATASOURCE_PASSWORD"] = "test"
+    properties["DATASOURCE_PASSWORD"] = "postgres"
```

Comment:
Kan du rulle tilbake denne endringen? Det er fint om vi har én standard for brukernavn/passord i postgres-testbasene våre, slik at vi slipper å tweake frem og tilbake for hver enkelt app vi jobber med. postgres/test har vært standard så langt i team familie.

Reply:
Det skal jeg fikse.